test/selectors/columnWidths-test.js

```diff
@@ -107,15 +107,15 @@ describe('columnWidths', function () {
       );
     assert.deepEqual(
       columnProps.map((column) => column.id),
-      [2, 4, 1, 3, 5, 7, 6]
+      [1, 2, 3, 4, 5, 6, 7]
```

Comment:
This test case was actually supposed to check if `columnProps` passes "should partition columns on fixed flag". But it incorrectly checked against a different order.

I fixed the test by making it compare against the partitioned order.